Modules/Scripted/Home/HomeLib/dependency_installer.py

```diff
@@ -56,7 +62,14 @@ def monai_pre_install():
     slicer.util._executePythonModule('light_the_torch', ['install', 'monai'])
 
 
+cudaVersion = os.popen("nvcc --version | egrep -o 'release [0-9\.]+' | egrep -o '[0-9\.]+' | sed -r -e 's/\.//g'").read().splitlines()[0]
```

Comment:
Upon launching LungAIR I got

```text
/bin/sh: 1: nvcc: not found
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/ebrahim/LungAIR-SuperBuild-testing-issue39/python-install/lib/python3.9/imp.py", line 169, in load_source
    module = _exec(spec, sys.modules[name])
  File "<frozen importlib._bootstrap>", line 613, in _exec
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/ebrahim/LungAIR-SuperBuild-testing-issue39/Slicer-build/lib/LungAIR-4.13/qt-scripted-modules/Home.py", line 7, in <module>
    from HomeLib import dependency_installer
  File "/home/ebrahim/LungAIR-SuperBuild-testing-issue39/Slicer-build/lib/LungAIR-4.13/qt-scripted-modules/HomeLib/dependency_installer.py", line 65, in <module>
    cudaVersion = os.popen("nvcc --version | egrep -o 'release [0-9\.]+' | egrep -o '[0-9\.]+' | sed -r -e 's/\.//g'").read().splitlines()[0]
IndexError: list index out of range
```

and then the Home module failed to load. It looks like nvcc is now a dependency of LungAIR? I don't think it should be, because we ought to be able to do everything without any nvidia hardware if we want to (i.e. run inference on CPU). 

In any case, we can't have the Home module crash-- the failure should be more graceful if `nvcc` does not exist.

Reply:
The `nvcc`-based code did not work and has been removed.